### PR TITLE
# Rename folders.

### PR DESCRIPTION
**Pull request by immmus**
Signed-off-by: immmus <fonto.trg@gmail.com>

### CODE REVIEW

File: open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/SubjectArea.java [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
 
-import org.odpi.openmetadata.accessservices.subjectarea.client.entities.glossaries.SubjectAreaGlossary;
-import org.odpi.openmetadata.accessservices.subjectarea.client.entities.terms.SubjectAreaTerm;
-import org.odpi.openmetadata.accessservices.subjectarea.client.entities.projects.SubjectAreaProject;
-import org.odpi.openmetadata.accessservices.subjectarea.client.entities.categories.SubjectAreaCategory;
+import org.odpi.openmetadata.accessservices.subjectarea.client.nodes.glossaries.SubjectAreaGlossary;
+import org.odpi.openmetadata.accessservices.subjectarea.client.nodes.terms.SubjectAreaTerm;
+import org.odpi.openmetadata.accessservices.subjectarea.client.nodes.projects.SubjectAreaProject;
+import org.odpi.openmetadata.accessservices.subjectarea.client.nodes.categories.SubjectAreaCategory;
 import org.odpi.openmetadata.accessservices.subjectarea.client.relationships.SubjectAreaRelationship;
```
Comment (davidradl) [inline open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/SubjectArea.java:9]: @immmus Did you want to change relationship to line as well?

File: open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/nodes/SubjectAreaNode.java [hunk 2]
```diff
@@ -105,9 +105,9 @@ public SubjectAreaNode(SubjectAreaRestClient subjectAreaRestClient) {
      * @return SubjectAreaEntityClient or null if this client is not present
```
Comment (davidradl) [inline open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/nodes/SubjectAreaNode.java:105]: Entity => Node

File: open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/nodes/terms/SubjectAreaTerm.java [hunk 3]
```diff
@@ -12,8 +12,8 @@
 public interface SubjectAreaTerm {
 
     /**
-     * @return {@link SubjectAreaEntityClient} for client calls(described in {@link SubjectAreaClient})
+     * @return {@link SubjectAreaNodeClient} for client calls(described in {@link SubjectAreaClient})
      * when working with Term objects
```
Comment (davidradl) [inline open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/nodes/terms/SubjectAreaTerm.java:16]: Term objects => Terms

File: open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/SubjectArea.java (hunk 1 above)
Comment (immmus) [reply open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/SubjectArea.java:9]: @davidradl No:)

File: open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/SubjectAreaNodeClient.java [hunk 4]
```diff
@@ -15,7 +15,7 @@
  * @param <E> inherited from {@link Node} object type
```
Comment (davidradl) [inline open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/SubjectAreaNodeClient.java:15]: remove object

File: open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/nodes/AbstractSubjectAreaEntity.java [hunk 5]
```diff
@@ -20,7 +20,7 @@
  * @param <E> inherited from {@link Node} object type
```
Comment (davidradl) [inline open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/nodes/AbstractSubjectAreaEntity.java:20]: remove object

File: open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/nodes/categories/SubjectAreaCategory.java [hunk 6]
```diff
@@ -13,14 +13,14 @@
 public interface SubjectAreaCategory {
 
     /**
-     * @return {@link SubjectAreaEntityClient} for client calls(described in {@link SubjectAreaClient})
+     * @return {@link SubjectAreaNodeClient} for client calls(described in {@link SubjectAreaClient})
      * when working with Category objects
```
Comment (davidradl) [inline open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/nodes/categories/SubjectAreaCategory.java:17]: @immmus in line with the Terms change ; Category objects => Categories

File: open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/nodes/categories/SubjectAreaCategory.java [hunk 7]
```diff
@@ -13,14 +13,14 @@
...
      */
-    SubjectAreaEntityClient<Category> category();
+    SubjectAreaNodeClient<Category> category();
 
     /**
-     * @return {@link SubjectAreaEntityClient} for client calls(described in {@link SubjectAreaClient})
+     * @return {@link SubjectAreaNodeClient} for client calls(described in {@link SubjectAreaClient})
      * when working with SubjectAreaDefinition objects
```
Comment (davidradl) [inline open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/nodes/categories/SubjectAreaCategory.java:23]: @immmus in line with the Terms change ; SubjectAreaDefinition objects => SubjectAreaDefinitions

**Comment by davidradl** [comment]
@immmus I am not sure if you noticed, I spotted some other inconsistencies after my last review.

**Comment by immmus** [comment]
> @immmus I am not sure if you noticed, I spotted some other inconsistencies after my last review.

Hi @davidradl, Yes, I saw, I'm just a little busy for now, I'll be back here a little later. Let's finish this for now pr -> https://github.com/odpi/egeria/pull/3366

**Comment by davidradl** [comment]
> > @immmus I am not sure if you noticed, I spotted some other inconsistencies after my last review.
> 
> Hi @davidradl, Yes, I saw, I'm just a little busy for now, I'll be back here a little later. Let's finish this for now pr -> #3366

Hi @immmus, sure - I have made review comments on that pr.

**Comment by immmus** [comment]
> > > @immmus I am not sure if you noticed, I spotted some other inconsistencies after my last review.
> > 
> > 
> > Hi @davidradl, Yes, I saw, I'm just a little busy for now, I'll be back here a little later. Let's finish this for now pr -> #3366
> 
> Hi @immmus, sure - I have made review comments on that pr.

I made the necessary changes yesterday

**Pull request merged**